Package/iOS/CMakeLists.txt

```diff
@@ -5,6 +5,14 @@ include(${CMAKE_CURRENT_SOURCE_DIR}/../../Apps/Playground/Playground/node_module
 set(PACKAGED_LIBS
     astc-codec
     astc
+	etc1
+	etc2
+	nvtt
+	squish
+	pvrtc
+	iqa
+	edtaa3
+	tinyexr
```

Comment:
These are additional libs we are now linking, right? If so they need to be added to gulpfile.js as well. I would definitely make sure to test the packages any time we add new dependencies.

Reply:
Done.